src/helper/LocalStorageHelper.js

```diff
@@ -0,0 +1,14 @@
+const TOKEN = "token";
+const USER = "user";
+
+export const setLocalStorageToken = (token) => localStorage.setItem(TOKEN, token);
+
+export const setLocalStorageUser = (user) => localStorage.setItem(USER, JSON.stringify(user));
+
+export const removeLocalStorageToken = () => localStorage.removeItem(TOKEN);
```

Comment:
Suggestion : You can have one remove function , in which you can remove both token and user as they are being used together